<!-- %Tag(FULL)%-->
<launch>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="talos_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="100" type="double"/>
  <param name="scale_angular" value="750" type="double"/>

  <node pkg="talos_teleop" type="talos_teleop_node" name="talos_teleop_node" output="screen"/>

</launch>
<!-- %EndTag(FULL)%-->
